firebase-firestore/src/main/java/com/google/firebase/firestore/FirebaseFirestore.java

```diff
@@ -362,6 +362,16 @@ public WriteBatch batch() {
     return shutdownInternal();
   }
 
+  /**
+   * Wait until all pending writes existed at the time of calling are sent to the backend.
+   *
+   * @return A {@link Task} which resolves when all pending writes are sent to the backend. If there
```

Comment:
Don't `@link` items that are already in the signature of the method. Use `@link` for anything related that's not part of the signature.

Use `{@code Task}` to highlight that it's a type without the extra visual noise that comes from an actual link.

Reply:
Done.